apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/test/java/gov/cms/bfd/pipeline/bridge/etl/McsTransformerIT.java

```diff
@@ -122,6 +156,11 @@ void shouldProduceExpectedClaimsAndDataSamples(
     }
   }
 
+  /**
+   * New first claim tests.
```

Comment:
This is the data for the parameterized tests; this is a good spot to describe what this test case is and what it's testing for.

Reply:
done